src/openforms/js/components/admin/form_design/variables/ServiceFetchConfigurationForm.js

```diff
@@ -62,6 +62,27 @@ const ServiceFetchConfigurationForm = ({formik, selectExisting = false}) => {
             }
             extraClassName="admin-fieldset"
           >
+            <FormRow>
+              <Field
+                name={'name'}
+                fieldBox
+                required
+                label={
+                  <FormattedMessage
+                    defaultMessage="Name"
+                    description="Service fetch configuration modal form name field label"
+                  />
+                }
+              >
+                <TextInput
+                  id="name"
+                  value={formik.values.name}
+                  maxLength="1000"
+                  {...formik.getFieldProps('name')}
+                />
+              </Field>
+            </FormRow>
+
             <FormRow>
               <Field
                 name={'method'}
```

Comment:
```suggestion
                name="method"
```
style convention with react (string) props -> just use double quotes